erpnext/accounts/utils.py

```diff
@@ -252,16 +252,14 @@ def add_ac(args=None):
 	if not ac.parent_account:
 		ac.parent_account = args.get("parent")
 
-	if getattr(ac, 'is_root', None):
+	if cint(ac.is_root):
 		ac.parent_account=''
 
 	ac.old_parent = ""
 	ac.freeze_account = "No"
 	if cint(ac.get("is_root")):
 		ac.parent_account = None
 		ac.flags.ignore_mandatory = True
-	else:
```

Comment:
Don't change here, instead pass is_root from js to fix the issue in charts.erpnext.com